pl_bolts/models/gans/__init__.py

```diff
@@ -1,7 +1,9 @@
 from pl_bolts.models.gans.basic.basic_gan_module import GAN  # noqa: F401
 from pl_bolts.models.gans.dcgan.dcgan_module import DCGAN  # noqa: F401
+from pl_bolts.models.gans.pix2pix.pix2pix_module import Pix2Pix
 
 __all__ = [
     "GAN",
     "DCGAN",
+    "Pix2Pix"
```

Comment:
Let's keep the comma at the end for the consistent `yapf` format.
```suggestion
    "Pix2Pix",
```

Reply:
I have done the mentioned change.

Reply:
Thank you for your swift action!